### PR TITLE
ant component and about us changes

### PR DESCRIPTION
**Pull request by fanimator12**
footer is not changed yet

### CODE REVIEW

File: ui/src/components/CERN_FOOTER.css [hunk 1]
```diff
@@ -22,22 +22,32 @@
   max-width: 1140px;
   width: 100%;
   margin: auto;
+  padding: 55px 3%;
```
Comment (drjova) [inline ui/src/components/CERN_FOOTER.css:25]: why do you need these? If we are using ant design should take care of it

Comment (drjova) [inline ui/src/components/CERN_FOOTER.css:25]: why do you need these? If we are using ant design should take care of it

File: ui/src/components/CERN_FOOTER.css [hunk 2]
```diff
@@ -22,22 +22,32 @@
...
 
-.row:before,
+/* .row:before,
 .row:after {
-  display: table;
-  content: " ";
-}
+    display: table;
+    content: " ";
+} */
```
Comment (drjova) [inline ui/src/components/CERN_FOOTER.css:50]: just delete them don't comment, if we want to recover it we can always go back to git history

Comment (drjova) [inline ui/src/components/CERN_FOOTER.css:50]: just delete them don't comment, if we want to recover it we can always go back to git history

File: ui/src/components/CERN_FOOTER.css [hunk 3]
```diff
@@ -81,23 +91,23 @@
...
+    display: block;
+    height: 2px;
+    width: 0%;
+    position: absolute;
+    bottom: 14px;
+    left: 0;
+    margin: 0 0 0 18px;
+    max-width: calc(100% - 18px);
+    background: #f1862e;
+} */
```
Comment (drjova) [inline ui/src/components/CERN_FOOTER.css:110]: please delete it

Comment (drjova) [inline ui/src/components/CERN_FOOTER.css:110]: please delete it

File: ui/src/components/CERN_FOOTER.css [hunk 4]
```diff
@@ -152,6 +162,12 @@
   padding-left: 0px;
   float: left;
   width: 66.6666666667%;
+  left: 0;
+  top: -15px;
+}
+
+#cern-footer .footer-third-col {
```
Comment (drjova) [inline ui/src/components/CERN_FOOTER.css:169]: I'm very confused, why are you modifying ant's grid? by default it should handle all of these

Comment (drjova) [inline ui/src/components/CERN_FOOTER.css:169]: I'm very confused, why are you modifying ant's grid? by default it should handle all of these

Comment (fanimator12) [reply ui/src/components/CERN_FOOTER.css]: I removed these and updated the ant grid

File: ui/src/components/CERN_FOOTER.css (hunk 1 above)
Comment (fanimator12) [reply ui/src/components/CERN_FOOTER.css]: removed

File: ui/src/components/AT_HEADER.css [hunk 5]
```diff
@@ -30,19 +36,36 @@
...
+  width: 100%;
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+  display: block;
+}
+
+.ant-menu-item {
   list-style: none;
+  float: right !important;
```
Comment (drjova) [inline ui/src/components/AT_HEADER.css:56]: does it work in smaller screens?

Comment (fanimator12) [reply ui/src/components/AT_HEADER.css:56]: Good call. On smaller screens, it doesn't work. I'll make a new commit once it's fixed.

**Pull request merged**